<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InpaintingSubWindowWidget</class>
 <widget class="QWidget" name="InpaintingSubWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InpaintingSubWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2,11">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="originsize_rb">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>以原图尺寸显示</string>
       </property>
       <property name="icon">
        <iconset resource="Inpainting.qrc">
         <normaloff>:/icons/icons/origin.ico</normaloff>:/icons/icons/origin.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="screensize_rb">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>以当前屏幕尺寸显示</string>
       </property>
       <property name="icon">
        <iconset resource="Inpainting.qrc">
         <normaloff>:/icons/icons/screen.ico</normaloff>:/icons/icons/screen.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>output image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="output_scroll">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>866</width>
          <height>739</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="output_image">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(240,240,240);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Inpainting.qrc"/>
 </resources>
 <connections/>
</ui>
